core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
@hachikuji Quick question: In your first comment above what do you mean by enabling (or turning on) both assignors? Aren't both range and round robin assignors already enabled for the new consumer to use? Thanks.